src/native/libs/ReadMe.md

```diff
@@ -0,0 +1,3 @@
+For information on how to build, how to update the binplacing of the native artifacts please refer to
+[How to building native components only](../../../docs/workflow/building/libraries/ReadMe.md#how-to-building-native-components-only)
```

Comment:
```suggestion
[How to building native components only](../../../docs/workflow/building/libraries/README.md#how-to-building-native-components-only)
```

Reply:
(the link won't work in the browser when the casing does not match)

Reply:
Thanks!